src/Header.jsx

```diff
@@ -10,7 +10,7 @@ const Header = () => {
   const { logout, isLoggedIn } = useContext(AuthContext);
 
   const handleLogout = () => {
-    fetch("http://localhost:24001/api/axion-auth-service/public/auth/logout", {
+    fetch(`${authServiceUrl}/public/auth/logout`, {
```

Comment:
I see import was fixed in SignInComponent, but not here. Having error when trying to log out:
![image](https://github.com/user-attachments/assets/1251d331-15c8-4eb4-9efb-925a3c41f05e)